.github/workflows/coveo-stew.yml

```diff
@@ -75,12 +75,23 @@ jobs:
         shell: bash
         run: pipx install "coveo-pypi-cli>=2.1.2"
 
-      - name: Publish to pypi
-        uses: ./.github/workflows/actions/publish-to-pypi
+      - name: Build
+        uses: ./.github/workflows/actions/build
+        id: build
         with:
           pre-release: ${{ github.ref != 'refs/heads/main' }}
           dry-run: ${{ github.ref != 'refs/heads/main' && github.event.inputs.publish != 'true' }}
 
+      - name: Publish to pypi
+        if: ${{ github.ref != 'refs/heads/main' && github.event.inputs.publish != 'true' }}
```

Comment:
I don't really understand this condition (we don't publish if we are outside of main) ?